Purchases/Misc/Obsoletions.swift

```diff
@@ -474,6 +474,25 @@ public extension Package {
     @objc var product: SKProduct { fatalError() }
 }
 
+public extension StoreProductDiscount.PaymentMode {
+    /// No payment mode specified
+    @available(iOS, obsoleted: 1, message: "This option no longer exists. PaymentMode would be nil instead.")
+    @available(tvOS, obsoleted: 1, message: "This option no longer exists. PaymentMode would be nil instead.")
+    @available(watchOS, obsoleted: 1, message: "This option no longer exists. PaymentMode would be nil instead.")
+    @available(macOS, obsoleted: 1, message: "This option no longer exists. PaymentMode would be nil instead.")
+    @available(macCatalyst, obsoleted: 1, message: "This option no longer exists. PaymentMode would be nil instead.")
+    static var none: StoreProductDiscount.PaymentMode { fatalError() }
+}
+
+// Note: `RCPaymentMode` is still available to Objective-C through `StoreProductDiscount.PaymentMode`.
```

Comment:
This worked to my surprise. Objective-C still works, and using `RCPaymentMode` from Swift now has the rename fix it.